src/ducks/_Table/index.js

```diff
@@ -39,40 +41,59 @@ const Table = ({
   itemProps,
   isLoading,
   getItemKey,
-  onRowClick
+  onRowClick,
+  ...props
 }) => (
   <table className={cx(styles.wrapper, className)}>
-    <thead>
-      <tr>
-        {columns.map(({ id, title, Title }) => (
-          <th key={id}>{Title ? <Title {...itemProps} /> : title}</th>
-        ))}
-      </tr>
-    </thead>
+    <DesktopOnly>
+      <thead>
+        <tr>
+          {columns.map(({ id, title, Title }) => (
+            <th key={id}>{Title ? <Title {...itemProps} /> : title}</th>
+          ))}
+        </tr>
+      </thead>
+    </DesktopOnly>
     <tbody>
-      {items.map((item, i) => {
-        const itemIndex = offset + i
-
-        return (
-          <tr
-            key={getItemKey ? getItemKey(item) : item[itemKeyProperty]}
-            onClick={onRowClick && (e => onRowClick(item, e))}
-          >
-            {columns.map(({ id, render, className }) => (
-              <td key={id} className={className}>
-                {render(item, itemProps, itemIndex)}
-              </td>
-            ))}
-          </tr>
-        )
-      })}
-      {minRowsPadding(minRows, columns, items)}
+      {!isLoading && items.length === 0 && (
+        <tr className={styles.disableHover}>
+          <td colSpan={columns.length}>
+            <EmptySection
+              wrapperClassName={styles.emptyWrapper}
+              type={BLOCKCHAIN_ADDRESS}
+              watchlist={props.watchlist}
+              refreshList={props.refreshList}
+            />
+          </td>
+        </tr>
+      )}
+      {items.length > 0 && (
+        <>
+          {items.map((item, i) => {
+            const itemIndex = offset + i
+            return (
+              <tr
+                key={getItemKey ? getItemKey(item) : item[itemKeyProperty]}
+                onClick={onRowClick && (e => onRowClick(item, e))}
+              >
+                {columns
+                  .filter(({ id }) =>
+                    props.isDesktop ? true : id !== 'CHECKBOX'
+                  )
```

Comment:
Columns' structure should be changed in the parent component that renders `Table` component.